app/controllers/comments_controller.rb

```diff
@@ -1,6 +1,14 @@
 class CommentsController < ApplicationController
   before_action :set_comment, only: %i[ show edit update destroy ]
-
+  before_action :is_authorized
+  include Pundit
```

Comment:
Not sure if it matters but add line 4 to the top

Reply:
Also, I think you can just put this once in the application controller

Reply:
thanks, I wasnt sure ab that so just in case i slapped it on all the controllers